ui/src/routes/_layout/organizations/$orgId/products/$productId/repositories/$repoId/runs/$runIndex/index.tsx

```diff
@@ -61,7 +61,30 @@ const RunComponent = () => {
       }),
   });
 
-  const downloadZipFile = async ({
+  const downloadLogs = async (runId: number) => {
```

Comment:
Commit message nits:
- "zipfile" -> "Zip file" (or "ZIP file")
- Please use passive voice, so no mention of "we" etc. as in "so we use default values of"
- IIUC there is no Scaleway-specific logic in this commit, so Scaleway should also not be mentioned as part of the commit message, to not create the false impression that this would only work on a Scaleway deployment.

Reply:
IIUC there actually is a Scaleway-specific issue with this.

Reply:
Like what?